src/process_utility.c

```diff
@@ -1491,6 +1491,19 @@ process_altertable_end_subcmd(Hypertable *ht, Node *parsetree, ObjectAddress *ob
 			Assert(IsA(cmd->def, ColumnDef));
 			process_alter_column_type_end(ht, cmd);
 			break;
+		case AT_EnableTrig:
+		case AT_EnableAlwaysTrig:
+		case AT_EnableReplicaTrig:
+		case AT_DisableTrig:
+		case AT_EnableTrigAll:
+		case AT_DisableTrigAll:
+		case AT_EnableTrigUser:
+		case AT_DisableTrigUser:
+			ereport(ERROR,
+					(errcode(ERRCODE_FEATURE_NOT_SUPPORTED),
+					 errmsg("Hypertables do not support adding "
```

Comment:
Error message is not following PG guidelines. (Should start non-capitalized.)

Reply:
Also, error message seems to imply we do not support "adding indexes" on hypertables. Is this true? Seems like a deal breaker?